packages/lesswrong/lib/collections/messages/collection.js

```diff
@@ -44,3 +45,18 @@ const Messages = createCollection({
 });
 
 export default Messages;
+
+makeEditable({
+  collection: Messages,
+  options: {
+    // Determines whether to use the comment editor configuration (e.g. Toolbars)
+    commentEditor: true,
+    // Determines whether to use the comment editor styles (e.g. Fonts)
+    commentStyles: true,
+    permissions: {
+      viewableBy: ['members'],
+      insertableBy: ['members'],
+      editableBy: Users.owns,
```

Comment:
No, `members` means everyone with an account, `Users.owns` means the owner of the document. In this case we use `members` because we already defined a document-level access-check that only allows the owners to edit the whole document.